src/test/java/de/bsi/secvisogram/csaf_cms_backend/rest/AdvisoryControllerTest.java

```diff
@@ -352,4 +362,157 @@ void createNewCsafDocumentVersionTest() {
+        when(advisoryService.getComments(advisoryId)).thenReturn(List.of(info));
+
+
+        String expected = String.format(
+                """
+                        [{
+                            "commentId": "%s",
+                            "advisoryId": "%s",
+                            "owner": "%s"
+                        }]
+                        """, commentId, advisoryId, owner
+        );
+
+        this.mockMvc.perform(get(commentRoute))
+                .andDo(print())
+                .andExpect(status().isOk())
+                .andExpect(content().json(expected));
+    }
+
+    @Test
+    void createCommentTest_invalidJson() throws Exception {
+
+        String invalidJson = "not a valid JSON string";
+
+        when(advisoryService.addComment(advisoryId, invalidJson)).thenThrow(JsonProcessingException.class);
+
+        this.mockMvc.perform(
+                        post(commentRoute).content(invalidJson).contentType(MediaType.APPLICATION_JSON).with(csrf()))
+                .andDo(print())
+                .andExpect(status().isBadRequest());
+    }
+
+
+    @Test
+    void createCommentTest_missingAdvisoryId() throws Exception {
+
+
+        String commentJson = """
+                {
+                    "commentText": "This is a comment."
+                }
+                """;
+
+        when(advisoryService.addComment(advisoryId, commentJson)).thenThrow(IllegalArgumentException.class);
+
+        this.mockMvc.perform(
+                        post(commentRoute).with(csrf()).content(commentJson).contentType(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest());
+    }
+
+
+    @Test
+    void createCommentTest() throws Exception {
+
+        IdAndRevision idRev = new IdAndRevision(UUID.randomUUID().toString(), "rev-123-abc");
+        String commentJson = """
+                {
+                    "commentText": "This is a comment.",
+                    "advisoryId": "some advisory ID we pretend exists."
+                }
+                """;
+
+        when(advisoryService.addComment(advisoryId, commentJson)).thenReturn(idRev);
+
+        String expected = String.format(
+                """
+                        {
+                            "id": "%s",
+                            "revision": "%s"
+                        }
+                        """, idRev.getId(), idRev.getRevision());
+
+        this.mockMvc.perform(
+                        post(commentRoute).with(csrf()).content(commentJson).contentType(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isCreated())
+                .andExpect(content().json(expected));
+
+    }
+
+
+    @Test
+    void changeCommentTest_notExisting() throws Exception {
+
+        doThrow(IdNotFoundException.class).when(advisoryService).updateComment(commentId, revision, commentText);
```

Comment:
Every test should have its own test fixture, so that it can be changed independently from other test. 
Its not necessary to make commentText and commentID static.

Reply:
I tried to reduce repeated code and reuse those values as the exact content doesn't really matter for the tests. I don't have a too strong opinion on that, so feel free to change this.